src/components/FileDownload/FileDownload.tsx

```diff
@@ -3,15 +3,19 @@ import cx from 'classnames';
 
 type FileDownloadProps = {
   className?: string;
+  documentSubType?: string;
+  documentType?: string;
   href: string;
   label?: string;
   name?: string;
   size?: string;
-  type?: string;
+  type: string;
```

Comment:
This doesn't cause any unintended 500 responses does it?

Reply:
I don't think so... this is just type-checking and isn't run on production anyway. 

I noticed that we're rendering the `type`  (PDF, XLS etc.) without a conditional, so updated the type declaration for it to be required